pom.xml

```diff
@@ -115,7 +115,7 @@
     <log4j2.version>2.17.2</log4j2.version>
     <slf4j.version>1.7.36</slf4j.version>
     <joda.version>2.9.9</joda.version>
-    <hadoop.version>2.10.1</hadoop.version>
+    <hadoop.version>2.10.2</hadoop.version>
```

Comment:
Avoid version upgrade in this PR?

Reply:
This release contains important security fixes for log4shell https://hadoop.apache.org/docs/r2.10.2/hadoop-project-dist/hadoop-common/release/2.10.2/RELEASENOTES.2.10.2.html 

Can you take up a task to patch it @yihua ?

Reply:
Got it.  I'll create a separate PR: HUDI-6977.

Reply:
Separate upgrade PR here: #9914